controllers/chat/deleteMessage.js

```diff
@@ -26,7 +26,9 @@ const deleteMessage = async (req, res) => {
     let baseMessage = {
       user_id: req.userId,
       message_type: 'deleted',
-      deleted_message_id: messageID
+      deleted_message_id: messageID,
+      deleted_message_created_at: message?.created_at,
+      deleted_message_updated_at: message?.updated_at
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The changes look good, but there's a potential issue here. If the `message` object is null or undefined, `message?.created_at` and `message?.updated_at` will be undefined. This could lead to unexpected behavior if the rest of your code doesn't handle these fields being undefined. 

To avoid this, you might want to provide default values for `deleted_message_created_at` and `deleted_message_updated_at` when `message` is null or undefined.

```diff
-      deleted_message_created_at: message?.created_at,
-      deleted_message_updated_at: message?.updated_at
+      deleted_message_created_at: message?.created_at || null,
+      deleted_message_updated_at: message?.updated_at || null
```
This way, if `message` is null or undefined, `deleted_message_created_at` and `deleted_message_updated_at` will be set to null instead of undefined.


<!-- This is an auto-generated comment by OSS CodeRabbit -->